safe-svg.php

```diff
@@ -728,6 +717,37 @@ protected function str_ends_with( $haystack, $needle ) {
 			return 0 === substr_compare( $haystack, $needle, -$len, $len );
 		}
 
+		/**
+		 * Set custom width or height of the SVG image.
```

Comment:
Since this method doesn't actually set anything, it returns these values, we should probably update this

Reply:
And thinking about it more, might be worth renaming this method to be `get_svg_dimensions` instead, or something similar